tap/api/api.go

```diff
@@ -103,6 +107,7 @@ type MizuEntry struct {
 	RequestSenderIp     string `json:"requestSenderIp" gorm:"column:requestSenderIp"`
 	Service             string `json:"service" gorm:"column:service"`
 	Timestamp           int64  `json:"timestamp" gorm:"column:timestamp"`
+	ElapsedTime         int64  `json:"elapsed_time" gorm:"column:elapsedTime"`
```

Comment:
I see others are in camelCase (in the json annotation), is there any reason it is different?)

Reply:
@gadotroee I'm sorry, I'm used to snake_case when I see a JSON field. Fixed it with https://github.com/up9inc/mizu/commit/516a3e59b062f864ad54f5d7ee3a4f8e4a8a2259